apps/server/src/modules/oauth/controller/oauth-sso.controller.spec.ts

```diff
@@ -11,24 +11,51 @@ import { AuthorizationParams } from './dto/authorization.params';
 import { OauthSSOController } from './oauth-sso.controller';
 
 describe('OAuthController', () => {
-	Configuration.set('HOST', 'https://mock.de');
+	let module: TestingModule;
 	let controller: OauthSSOController;
-	let oauthUc: OauthUc;
+	let oauthUc: DeepMocked<OauthUc>;
+
+	const mockHost = 'https://mock.de';
 	const defaultJWT =
 		'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyLCJ1dWlkIjoiMTIzIn0.H_iI0kYNrlAUtHfP2Db0EmDs4cH2SV9W-p7EU4K24bI';
```

Comment:
For integration and unit test i think we can replace the real jwt with any string like 'JWT_TOKEN' or so.

Reply:
done